app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km: float = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | Distance) -> Distance | float | int:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: int | Distance) -> Distance | float | int:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
+        return self
+
+    def __mul__(self, scalar: int | float) -> Distance | int | float:
+        if isinstance(scalar, (int, float)):
+            return Distance(self.km * scalar)
+
+    def __truediv__(self, scalar: int | float) -> Distance | float:
+        if isinstance(scalar, (int, float)):
+            return Distance(round(self.km / scalar, 2))
+
+    def __lt__(self, other: int | Distance | float) -> bool:
```

Comment:
Check this tool for working with comparison magic methods
https://docs.python.org/3/library/functools.html#functools.total_ordering